satellite/accounting/live/plainmemory.go

```diff
@@ -76,28 +40,32 @@ func (pmac *plainMemoryLiveAccounting) GetProjectStorageUsage(ctx context.Contex
 	pmac.spaceMapLock.Lock()
 	defer pmac.spaceMapLock.Unlock()
 	curVal := pmac.spaceDeltas[projectID]
-	return curVal.inlineSpace, curVal.remoteSpace, nil
+	return curVal.InlineSpace, curVal.RemoteSpace, nil
 }
 
 // AddProjectStorageUsage lets the live accounting know that the given
-// project has just added inlineSpaceUsed bytes of inline space usage
-// and remoteSpaceUsed bytes of remote space usage.
+// project has just added InlineSpaceUsed bytes of inline space usage
+// and RemoteSpaceUsed bytes of remote space usage.
 func (pmac *plainMemoryLiveAccounting) AddProjectStorageUsage(ctx context.Context, projectID uuid.UUID, inlineSpaceUsed, remoteSpaceUsed int64) error {
 	pmac.spaceMapLock.Lock()
 	defer pmac.spaceMapLock.Unlock()
 	curVal := pmac.spaceDeltas[projectID]
-	curVal.inlineSpace += inlineSpaceUsed
-	curVal.remoteSpace += remoteSpaceUsed
+	curVal.InlineSpace += inlineSpaceUsed
+	curVal.RemoteSpace += remoteSpaceUsed
 	pmac.spaceDeltas[projectID] = curVal
 	return nil
 }
 
 // ResetTotals reset all space-used totals for all projects back to zero. This
 // would normally be done in concert with calculating new tally counts in the
 // accountingDB.
-func (pmac *plainMemoryLiveAccounting) ResetTotals() {
+func (pmac *plainMemoryLiveAccounting) ResetTotals(ctx context.Context) error {
 	pmac.log.Info("Resetting real-time accounting data")
```

Comment:
I would change this to a `Debug` in this branch although this line has not been modified in this branch.